Galaxy init no rest

alikins opened the pull request:
##### ISSUE TYPE
- Bugfix Pull Request
##### COMPONENT NAME

lib/ansible/cli/galaxy.py
##### ANSIBLE VERSION

```
ansible 2.3.0 (galaxy_init_no_rest 1ff3c83863) last updated 2016/10/19 15:08:43 (GMT -400)
  lib/ansible/modules/core: (detached HEAD 6145e24ed4) last updated 2016/10/19 12:31:40 (GMT -400)
  lib/ansible/modules/extras: (detached HEAD accb04b867) last updated 2016/10/19 12:31:41 (GMT -400)
  config file = /home/adrian/.ansible.cfg
  configured module search path = Default w/o overrides
```
##### SUMMARY

Remove platforms api request from galaxy 'init'

Instead of populating the platform comment in the default meta/main.yml with a dict of platforms fetch from the galaxy REST api, just use the default now included in the template.

Fixes #18103

chouseknecht commented:
@alikins: I think a better approach would be to modify Galaxy and change how platforms are entered. I think we can eliminate the need for surfacing a list of platforms altogether, thus eliminating the API call.  Here is my proposal: https://github.com/ansible/galaxy-issues/issues/206

bcoca commented:
I agree, we should avoid populating platforms aside from a couple of examples. Most people just ignore or delete 99% of the entries anyways.

alikins committed:
Remove platforms api request from galaxy 'init'

Instead of populating the platform comment in the default
meta/main.yml with a dict of platforms fetch from the
galaxy REST API, this removes the API call and adds a
commented out example platforms list to the default meta/main.yml

Fixes #18103
